exchange/cryptopia.co.nz/api.go

```diff
@@ -219,10 +220,10 @@ func getMarketOrderGroups(count int, markets ...string) ([]MarketOrdersWithLabel
 	}
 	resp, err := requestGet("getmarketordergroups", requestParams)
 	if err != nil {
-		return nil, errors.Wrapf(err, "GetMarketOrderGroups failed, markets: %s", strings.Join(markets, " "))
+		return nil, fmt.Errorf("GetMarketOrderGroups failed, markets: %s", strings.Join(markets, " "))
```

Comment:
Put the original error message into the format string for all of the formerly Wrapf calls

Reply:
https://github.com/skycoin/exchange-api/pull/18/commits/8c61cc882a298c312243df8b71e1df211c345c8b

Done.